components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint.util/src/main/java/org/wso2/carbon/identity/application/authentication/endpoint/util/Constants.java

```diff
@@ -279,6 +279,10 @@ public static class ErrorToi18nMappingConstants {
                 "authentication.flow.timeout_authentication.flow.timeout.description";
         public static final String AUTHENTICATION_FLOW_TIMEOUT_AUTHENTICATION_FLOW_TIMEOUT_DESCRIPTION_I18N_KEY =
                 "authentication.flow.timeout";
+        public static final String INVALID_RESPONSE_TYPE_FOR_QUERY_JWT_DESCRIPTION =
```

Comment:
For the description, we seems have used dots between words. Can we follow the same pattern?

Reply:
since invalid_request is already coming from the OAuthConstants, thought of adding the rest of the description separated by _s. We can change it to `invalid_request_invalid.response.type.for.query.jwt`. Is that okay?

Reply:
In that case, from ouath side, error code and the description would be as follows:
error_code = invalid_request
error_description(suberror_code) = invalid.response.type.for.query.jwt
In the util method, it will be joined with a _.

Reply:
Addressed this with https://github.com/wso2/carbon-identity-framework/pull/5011/commits/6107540b4a712c6749981b8c6d62ddf67ae5a030 
to align with the existing descriptions.
https://github.com/wso2/carbon-identity-framework/blob/dfa3965c725b497e1450abce2b35e370209bee4d/components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint.util/src/main/java/org/wso2/carbon/identity/application/authentication/endpoint/util/Constants.java#L147